--- a/gcc/config/tms9900/tms9900.h	2026-05-02 00:00:00.000000000 +0000
+++ b/gcc/config/tms9900/tms9900.h	2026-05-02 00:00:00.000000000 +0000
@@ -615,25 +615,16 @@ typedef struct tms9900_args
   int named_count;  /* Number of named arguments (for varargs) */  
 } CUMULATIVE_ARGS;
 
-extern int tms9900_function_arg_padding (enum machine_mode mode,
-                                         const_tree type);
-extern void tms9900_function_arg_advance (CUMULATIVE_ARGS *cum,
-                                          enum machine_mode mode,
-                                          tree type,
-                                          int named);
-extern void tms9900_init_cumulative_args (CUMULATIVE_ARGS *cum,
-                                          tree fntype,
-                                          rtx libname);
-extern rtx tms9900_function_arg (CUMULATIVE_ARGS *cum,
-                                 enum machine_mode mode,
-                                 tree type,
-                                 int named);
-extern int tms9900_initial_elimination_offset (int from, int to);
-extern int tms9900_address_type (rtx op, enum machine_mode mode);
-extern int tms9900_reg_ok_for_base (int strict, rtx reg);
-extern int tms9900_go_if_legitimate_address (enum machine_mode mode, rtx operand, int strict);
-extern void tms9900_output_ascii (FILE *stream, const char *ptr, int len);
-extern bool tms9900_legitimate_constant_p (rtx x);
-extern bool tms9900_constant_address_p (rtx x);
+extern int tms9900_function_arg_padding ();
+extern void tms9900_function_arg_advance ();
+extern void tms9900_init_cumulative_args ();
+extern rtx tms9900_function_arg ();
+extern int tms9900_initial_elimination_offset ();
+extern int tms9900_address_type ();
+extern int tms9900_reg_ok_for_base ();
+extern int tms9900_go_if_legitimate_address ();
+extern void tms9900_output_ascii ();
+extern bool tms9900_legitimate_constant_p ();
+extern bool tms9900_constant_address_p ();
 
 /* If defined, a C expression which determines whether, and in which direction,
