--- a/gcc/config/tms9900/tms9900.h	2026-05-02 00:00:00.000000000 +0000
+++ b/gcc/config/tms9900/tms9900.h	2026-05-02 00:00:00.000000000 +0000
@@ -627,8 +627,10 @@ extern int tms9900_go_if_legitimate_address ();
 extern int tms9900_go_if_legitimate_address ();
 extern void tms9900_output_ascii ();
+#ifndef IN_LIBGCC2
 extern bool tms9900_legitimate_constant_p ();
 extern bool tms9900_constant_address_p ();
+#endif
 
 extern void print_operand_address ();
 extern int legitimate_address_p ();
 extern void notice_update_cc_on_set ();
@@ -1257,8 +1259,10 @@
   } while (0)
 
 
+#ifndef IN_LIBGCC2
 void tms9900_asm_output_dwarf_delta (FILE *file, int size,
                                const char *lab1, const char *lab2);
 void tms9900_asm_output_dwarf_offset (FILE *file, int size, const char *lab,
                                 section *base);
+#endif
